### PR TITLE
Installed uploading file

### PR DESCRIPTION
**Pull request by SaranchukVadym**

### CODE REVIEW

File: sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        - dropColumn:
+            columnName: RSM_PATH
+            tableName: RESUME
+
+  - changeSet:
+      id: 03.05.2021_drop_not_null_constraint_of_RSM_EXT_column_in_resume_table
+      author: Saranchuk Vadym
+      preConditions:
+        onFail: MARK_RAN
+        columnExists:
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml:20]: Need check notnull constraint exists

Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml:20]: Need check notnull constraint exists

File: sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            columnName: RSM_EXT
+            tableName: RESUME
+
+  - changeSet:
+      id: 03.05.2021_drop_not_null_constraint_of_RSM_SIZE_column_in_resume_table
+      author: Saranchuk Vadym
+      preConditions:
+        onFail: MARK_RAN
+        columnExists:
+          tableName: RESUME
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml:35]: the same

File: sandbox-backend/src/main/resources/db/changelog/changes/05.05.2021_modify_data_type_of_CN_PHONE_column_in_candidate_table.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            newDataType: VARCHAR(17)
+            tableName: CANDIDATE
+
+  - changeSet:
+      id: 05.05.2021_add_not_null_constraint_into_CN_PHONE_column_in_candidate_table
+      author: Saranchuk Vadym
+      preConditions:
+        onFail: MARK_RAN
+        columnExists:
+          tableName: CANDIDATE
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/05.05.2021_modify_data_type_of_CN_PHONE_column_in_candidate_table.yaml:22]: Need to check is notnull constraint not exists

File: sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml (hunk 1 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml:20]: corrected

Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml:20]: corrected

File: sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml (hunk 2 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/03.05.2021_modify_resume_table.yaml:35]: corrected

File: sandbox-backend/src/main/resources/db/changelog/changes/05.05.2021_modify_data_type_of_CN_PHONE_column_in_candidate_table.yaml (hunk 3 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/05.05.2021_modify_data_type_of_CN_PHONE_column_in_candidate_table.yaml:22]: corrected

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/FileUploadServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+import java.net.URLDecoder;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+@Qualifier("fileUploadServiceImpl")
```
Comment (ikantsevich) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/FileUploadServiceImpl.java:35]: Do we need Qualifier in this place?

Comment (SaranchukVadym) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/FileUploadServiceImpl.java]: No, corrected

**Review by ikantsevich — APPROVED** [review]

**Pull request merged**